### PR TITLE
Path

### PR DESCRIPTION
**Pull request by Alannables**
Fixed auto conflict between controller driving and auto path

### CODE REVIEW

File: .pathplanner/settings.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
+<<<<<<< HEAD
```
Comment (RobertTheBlair) [inline .pathplanner/settings.json]: Need to resolve the conflict

File: photonvision_config/cameras/HP_TrueVision_HD_Camera/config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (RobertTheBlair) [inline photonvision_config/cameras/HP_TrueVision_HD_Camera/config.json]: Consider removing these unused photonvision

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -152,11 +152,11 @@ public static class VisionConstants {
   public static class AutoConstants {
     public static TrapezoidProfile.Constraints THETA_CONSTRAINTS =
         new TrapezoidProfile.Constraints(PI, 2 / PI);
-    public static double THETA_kP = 1.2;
+    public static double THETA_kP = 1.2; // 1.2 original
     public static double THETA_kI = 0.0;
     public static double THETA_kD = 0.0;
 
-    public static double X_kP = 1.2;
+    public static double X_kP = 0.1; // 1.2 original
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/Constants.java:159]: Adjust on real robot

**Review by RobertTheBlair — CHANGES_REQUESTED** [review]
Needed a few changes

**Review by Alannables — COMMENTED** [review]
Good

**Review by RobertTheBlair — APPROVED** [review]

**Pull request merged**